### DISCUSSION TITLE
Listen for event from external process ?

### DISCUSSION THREAD

**Discussion by DEADF00D** [miguelgrinberg/Flask-SocketIO#1800] [Q&A] [answered]
As you can emit from an external process with:
```
sio = SocketIO(message_queue='redis://')
sio.emit('hello', 'world')
```
I was wondering if you could also listen for event from an external process like so:
```
@sio.on('hello')
def cb(data):
    print(data)
```
I tried to do it this way, without success.  
Does anyone achieved to make this work ?

**Comment by miguelgrinberg** [comment, marked as answer]
This is not supported, sorry.

**Reply by DEADF00D** [reply]
Hi Miguel,
Thanks for your answer.
